django_project/trail/models/point_of_interest.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
+"""Model definitions for a trail app"""
+
+import os
+import uuid as uuid_lib
+
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.text import slugify
+from django.conf.global_settings import MEDIA_ROOT
+
+from unidecode import unidecode
+
+from core.settings.contrib import STOP_WORDS
+from category import Category
+from trail_section import TrailSection
+
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '01/10/2018'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+def increment_slug(_name):
+    """Function to increment slug.
+
+    If there already exists the trail name and colour,
+    the slug will be incremented
+    e.g. trail-#25HHfff, trail-#25HHfff etc.
+    """
+
+    existing_pois = POI.objects.all()
+    new_poi_name = '%s' % (_name)
+    for poi in existing_pois:
+        if _name == poi.name:
+            _count_names = POI.objects.filter(
+                name=_name)
+            count = _count_names.count() + 1
+            new_poi_name = '%s %s' % (_name, count)
+            break
+
+    return new_poi_name
+
+
+class POI(models.Model):
+    "Model definition for a Point of Interest (POI)."
+
+    name = models.CharField(
+        _('Name of Point of Interest(POI)'),
+        max_length = 255,
+        null = True,
+        blank = True,
```

Comment:
above in a CharField definition you specify null=False, blank=False here you specify them as True. Please only specify when you are _changing the default_.